nightfall-optimist/src/services/block-utils.mjs

```diff
@@ -5,7 +5,7 @@ import Web3 from 'web3';
 // of the block object isn't.  They can thus be called directly when instantiating the Block class
 // would be problematic because of its reliance on the Optimist database.
 
-const { BLOCK_TYPES } = constants;
+const { SIGNATURES } = constants;
```

Comment:
Signatures variable is stored in config and not in constants
```suggestion
const { SIGNATURES } = config;
```

Reply:
done